conf/locale/locale_en-US.ini

```diff
@@ -510,9 +510,9 @@ pulls.merged = Merged
 pulls.has_merged = This pull request has been merged successfully!
 pulls.data_broken = Data of this pull request has been broken due to deletion of fork information.
 pulls.is_checking = The conflict checking is still in progress, please refresh page in few moments.
-pulls.can_auto_merge_desc = You can perform auto-merge operation on this pull request.
-pulls.cannot_auto_merge_desc = You can't perform auto-merge operation because there are conflicts between commits.
-pulls.cannot_auto_merge_helper = Please use command line tool to solve it.
+pulls.can_auto_merge_desc = This pull request can be automatically merged.
```

Comment:
Yes, you're right, I accidentally switched "automatically" and "merged". I'll update